fms/models/hf/utils.py

```diff
@@ -207,6 +207,24 @@ def _map_model_config(architecture, config):
         config_params["head_dim"] = getattr(
             config, "head_dim", config.hidden_size // config.num_attention_heads
         )
+    elif architecture == "GraniteMoeHybridForCausalLM":
+        inner_dim = config.intermediate_size
```

Comment:
inner_dim is computed and never used. We can use it to sanity-check or derive hidden_grow_factor from intermediate_size:
`hidden_grow_factor = config.intermediate_size / config.hidden_size
`